core/src/main/java/pl/agh/edu/ui/component/button/DifficultyButton.java

```diff
@@ -0,0 +1,54 @@
+package pl.agh.edu.ui.component.button;
+
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.engine.hotel.dificulty.DifficultyLevel;
+import pl.agh.edu.ui.language.LanguageChangeListener;
+import pl.agh.edu.ui.language.LanguageManager;
+import pl.agh.edu.ui.resolution.ResolutionChangeListener;
+import pl.agh.edu.ui.resolution.ResolutionManager;
+
+public class DifficultyButton extends TextButton implements LanguageChangeListener, ResolutionChangeListener {
+	public final DifficultyLevel difficulty;
+	public final Skin skin;
+
+	public DifficultyButton(DifficultyLevel difficulty, Skin skin) {
```

Comment:
trzoszkę wygodniej było by aby skin był z automatu ładowany a nie podawany za każdym razem w konstruktorze

Reply:
tak tak miałem to poprawić